tests/test_client.py

```diff
@@ -365,7 +365,7 @@ def raise_it(exc_info):
         reraise(*exc_info)
 
     sentry_init(ignore_errors=[ZeroDivisionError], transport=_TestTransport())
-    Hub.current._capture_internal_exception = raise_it
+    Hub.current.scope._capture_internal_exception = raise_it
```

Comment:
I duplicated the internal function `_capture_internal_exception` from Hub into Scope. So the one in Scope is called here, so we also need to apply our function to check if was called also on the Scope. As this is an internal function this is not a breaking change and fine to release in a minor version.